docs/index.rst

```diff
@@ -0,0 +1,95 @@
+PreREISE
+=========
+This page is dedicated to the collection, processing and formatting of data that are
+fed to the simulation engine. PreREISE is an open source package written in Python that
+is available on `GitHub <https://github.com/Breakthrough-Energy/PreREISE>`_.
+
+The generation of input data rely on the grid model. For this reason, you must have
+`PowerSimData <https://github.com/Breakthrough-Energy/PowerSimData>`_ cloned in a
+folder adjacent to your clone of PreREISE.
+
+
+General Comments
+----------------
+Profiles are CSV (Comma Separated Values) files enclosing a series of data points
+indexed in time orders. The simulation engine expects a demand, hydro, solar and wind
+profile. All the profiles have UTC timestamp as indices. Columns are the zone IDs and
+plant IDs for the demand and hydro/solar/wind profiles, respectively. Values are load in
+MWh in each zone for the demand and energy generated by a 1MW generator for hydro,
+solar and wind profiles.
+
+The **[type]_v[D]** naming convention is used, where:
+
++ **type** specifies the profile type (demand, hydro, solar or wind) and
++ **D** refers to the date the file has been created using the 3 letters Month (e.g.,
+  Jan) and 4 digits Year (e.g., 2021) format.
+
+If two or more profiles of the same type are created the same month of the same year,
+the different versions will be differentiated using a lowercase letter added right
+after the year, e.g, ``demand_vJan2021.csv`` (first), ``demand_vJan2021b.csv`` (second),
+``demand_vJan2021c.csv`` (third) and so forth.
+
+.. note::
+    Input data are publicly available for the Breakthrough Energy Sciences (BES) grid
+    model (see `publication <https://arxiv.org/abs/2002.06155>`_). You don't need to
+    generate any input data if you use the scenario framework to carry out power flow
+    study.
```

Comment:
This is nice. Is there a way to emphasis this note? According the the feedbacks from our collaborators, there is a general confusion: one needs to generate time series profiles via `PreREISE` before running simulations.

Reply:
It looks like this now:
<img width="810" alt="Screen Shot 2021-07-21 at 4 23 21 PM" src="https://user-images.githubusercontent.com/16549571/126572111-2546df0a-9de2-43ca-b55a-fc0367baf91c.png">